ray_provider/operators/anyscale_job.py

```diff
@@ -0,0 +1,143 @@
+
+from ray_provider.operators.base import AnyscaleBaseOperator
+from ray_provider.sensors.anyscale_job import AnyscaleProductionJobSensor
+
+from anyscale.shared_anyscale_utils.utils.byod import BYODInfo
+from anyscale.sdk.anyscale_client.models.create_production_job import CreateProductionJob
+
+_POKE_INTERVAL = 60
+
+
+class AnyscaleCreateProductionJobOperator(AnyscaleBaseOperator):
+    """
+    An operator that creates an Production Job.
+    :param name: Name of the job. (templated)
+    :param project_id: Id of the project this job will start clusters in. (templated)
+    :param entrypoint: A script that will be run to start your job.
+        This command will be run in the root directory of
+        the specified runtime env. Eg. 'python script.py' (templated)
+    :param build_id: The id of the cluster env build.
+        This id will determine the docker image your job is run on. (templated)
+    :param docker: Docker image for BYOD. (templated)
+    :param max_retries: The number of retries this job will attempt on failure.
+        Set to None to set infinite retries. (templated)
+    :param description: Description of the job. (templated)
+    :param runtime_env: A ray runtime env json. (templated)
+        Your entrypoint will be run in the environment specified by this runtime env. (templated)
+    :param compute_config_id: The id of the compute configuration that you want to use.
+        This id will specify the resources required for your job. (templated)
+    :param ray_version: Ray version (only used for BYOD). (templated)
+    :param python_version: Python version (only used for BYOD). (templated)
+    :param wait_for_completion: If True, waits for creation of the cluster to complete. (default: True)
+    """
+
+    template_fields: Sequence[str] = [
+        "name",
+        "auth_token",
+        "project_id",
+        "entrypoint",
+        "build_id",
+        "docker",
+        "description",
+        "runtime_env",
+        "compute_config_id",
+        "ray_version",
+        "python_version",
+    ]
+
+    def __init__(
+        self,
+        name: str,
+        project_id: str,
+        entrypoint: str,
+        build_id: str = None,
+        docker: str = None,
+        max_retries: int = 1,
+        description: str = None,
+        runtime_env: dict = None,
+        compute_config_id: str = None,
+        ray_version: Optional[str] = None,
+        python_version: Optional[str] = None,
+        wait_for_completion: Optional[bool] = True,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+
+        self.name = name
+        self.docker = docker
+        self.entrypoint = entrypoint
+        self.project_id = project_id
+        self.description = description
+        self.max_retries = max_retries
+        self.runtime_env = runtime_env
+        self.compute_config_id = compute_config_id
+        self.ray_version = ray_version or "1.13.0"
+        self.python_version = python_version or "py38"
```

Comment:
We could use these as default values also when getting these params.
WDYT?

Reply:
Can you also update in other places as well?

Reply:
Yes, I updated all the operators.